src/test/java/org/bookstore/service/BookServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package org.bookstore.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.math.BigDecimal;
+import java.util.Set;
+import org.bookstore.dto.book.BookDto;
+import org.bookstore.dto.book.CreateBookRequestDto;
+import org.bookstore.mapper.BookMapper;
+import org.bookstore.mapper.impl.BookMapperImpl;
+import org.bookstore.model.Book;
+import org.bookstore.model.Category;
+import org.bookstore.repository.BookRepository;
+import org.bookstore.repository.CategoryRepository;
+import org.bookstore.service.impl.BookServiceImpl;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class BookServiceTest {
```

Comment:
Where is test cases for other methods in service?

Reply:
@viacheslav-torbin you didn't answer :)